cmake/modules/RootNewMacros.cmake

```diff
@@ -1079,6 +1079,22 @@ function(ROOT_ADD_TEST test)
 endfunction()
 
 #----------------------------------------------------------------------------
+# function ROOT_ADD_GTEST(<testsuite> <name> LIBRARIES)
+#
+function(ROOT_ADD_GTEST test_suite test_name)
```

Comment:
How do I specify multiple sources? Can you add this to the doc of the CMake function? (I don't see how binary name and source files map to `<testsuite>` and `<name>`).